### DISCUSSION TITLE
Drag & drop: customize target with CSS

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#817] [Q&A] [answered]
Hi

When dragging an item from a list, I'm able to 
- customize initial state (when item is clicked)
- during drag

but, I would like to customize the target of the drop, before I drop it there 

Currently, we can see some grey text under the dragged element

[https://codepen.io/GATIGA/pen/PoJrJQq](https://codepen.io/GATIGA/pen/PoJrJQq)

Does anybody have an idea on how I can customize it? 

(I've been trying almost all selectors I could find [https://www.w3schools.com/cssref/css_selectors.asp](https://www.w3schools.com/cssref/css_selectors.asp)

Regards

**Comment by GalinhaLX** [comment]
Also have a strange behavior:
while dragging item, the height of the "box" is bigger, even if I set all sizes, any idea why this happens?

**Comment by DmitrySharabin** [comment, marked as answer]
Since Mavo uses [Dragula](https://bevacqua.github.io/dragula/) under the hood, I would suggest you take a look at [the classes it exposes](https://github.com/bevacqua/dragula#css). That might help.

Here are the styles Mavo supports out-of-box (that you can override, of course 😄):

https://github.com/mavoweb/mavo/blob/4b719aea26a4072fb193fe5c068cf894e88a527e/src-css/mavo.scss#L148-L175

**Reply by GalinhaLX** [reply]
Thank you @DmitrySharabin !

I don't know why... I was looking at draggable.js :(
